spec/wowza_rest/smils_spec.rb

```diff
@@ -0,0 +1,197 @@
+require 'spec_helper'
+
+RSpec.describe WowzaRest::Client do
```

Comment:
this should be `WowzaRest::SMILs` and replace `described_class` below with client namespace